drivers/scsi/mac_esp.c:	if (MACHW_PRESENT(MAC_SCSI_96)) {
drivers/scsi/mac_esp.c:	if (MACHW_PRESENT(MAC_SCSI_96_2)) {
include/asm-m68k/machw.h:#define MACHW_PRESENT(name)	(mac_hw_present.name)
---
 arch/m68k/mac/platform.c |   10 ++++++++++
 1 file changed, 10 insertions(+)

--- /dev/null
+++ b/arch/m68k/mac/platform.c
@@ -0,0 +1,10 @@
+/*
+ *  Copyright (C) 2007 Geert Uytterhoeven
+ *
+ * This file is subject to the terms and conditions of the GNU General Public
+ * License.  See the file COPYING in the main directory of this archive
+ * for more details.
+ */
+
+#include <linux/platform_device.h>
+
